Example.xcodeproj/project.pbxproj

```diff
@@ -376,7 +392,7 @@
 					"@executable_path/Frameworks",
 				);
 				MARKETING_VERSION = 1.0;
-				PRODUCT_BUNDLE_IDENTIFIER = com.phrase.Example;
+				PRODUCT_BUNDLE_IDENTIFIER = de.apploft.AppTemplate;
```

Comment:
Probably the identifier should not change here?

Reply:
Yes, I'll change that back.